eel_hole/models.py

```diff
@@ -20,6 +20,9 @@ class User(UserMixin, db.Model):
     auth0_id: Mapped[str] = mapped_column(unique=True)
     username: Mapped[str]
     email: Mapped[str]
+    accepted_privacy_policy: Mapped[bool] = mapped_column(default=False)
+    do_individual_outreach: Mapped[bool] = mapped_column(default=False)
+    send_newsletter: Mapped[bool] = mapped_column(default=False)
```

Comment:
By making this a setting, will users believe that changing it from "true" to "false" will unsubscribe them from the newsletter?